optaplanner-examples/src/main/resources/org/optaplanner/examples/vehiclerouting/vehicleRoutingSolverConfig.xml

```diff
@@ -71,7 +71,17 @@
           </nearbySelection>
         </secondarySubListSelector>
       </subListSwapMoveSelector>
-      <kOptListMoveSelector/>
+      <kOptListMoveSelector>
+        <originSelector id="5"/>
+        <valueSelector>
+          <nearbySelection>
+            <originValueSelector mimicSelectorRef="5"/>
+            <nearbyDistanceMeterClass>org.optaplanner.examples.vehiclerouting.domain.solver.nearby.CustomerNearbyDistanceMeter</nearbyDistanceMeterClass>
+            <nearbySelectionDistributionType>PARABOLIC_DISTRIBUTION</nearbySelectionDistributionType>
+            <parabolicDistributionSizeMaximum>40</parabolicDistributionSizeMaximum>
+          </nearbySelection>
+        </valueSelector>
+      </kOptListMoveSelector>
```

Comment:
Before we enable (or even merge) this, can we please do some benchmarks on whether or not this is beneficial?

Pick three data sets; small, medium, large. Let it run for, say, 1 minute unimproved with both nearby enabled and disabled. And let's compare those runs.

Reply:
patchCycles and route endpoints can be used now, so all the old moves are now possible (although there are probably many more new moves than old moves).